### DISCUSSION TITLE
Sort / filter tasks by reminder / alarm date-time

### DISCUSSION THREAD

**Discussion by milkywade** [TechbeeAT/jtxBoard#1587] [Ideas]
On jtx board there is no group nor sort function that can use the reminder/alarm date-time.

This is an issue for me as I have hundreds of tasks and only fill in the reminder time, never the due date (this works for me and saves me time).

On iOS, such filter is possible and actually already built-in: there is a "today" filter that shows only tasks with a reminder/alarm date-time set to today.

On Outlook (desktop client), I just sort tasks by the reminder time and thus know which are the next due ones.

On jtx board, I just have hundreds of tasks that I can not tell apart from each other.

**Comment by patrickunterwegs** [comment]
Hi @milkywade , 
I'm not sure if your request is so easy to be done. 
If the list is sorted by the alarm date, what date should be considered when there are multiple alarms? The first one? The next one? The last one? Then how would that change if you sort ascending or descending? So this would get rather tricky. 

In your case I would rather recommend you to use the start or the due date. You can then also activate the option to always create an alarm on start or on due, then it is always clear and you can sort and filter the list by start date or due date!

**Comment by milkywade** [comment]
The less fields to input the better. Today im using ios or outlook and i only have to inout the alarm. If to use another product i have to Input as well the start date, this is a deal killer to me. Its not just the task creation, its also than when postponing the task, you again have to change things in two fields

--

**Reply by patrickunterwegs** [reply]
But what would be your answer to my questions? What would be the behavior of the app in unclear cases?

By the way did you leave the bad review on Google Play?

**Comment by milkywade** [comment]
Hi Patrick, thank you for your interest on this feature. To your requested questions:

“If the list is sorted by the alarm date, what date should be considered when there are multiple alarms? The first one? The next one? The last one?”

The closest one in time to now().

“Then how would that change if you sort ascending or descending? So this would get rather tricky.”

Use answer to above to sort asc or desc.

Let me know other questions.
